lib/buildkite/test_collector/minitest_plugin/trace.rb

```diff
@@ -72,17 +70,20 @@ def project_dir
     end
 
     def failure_reason
-      @failure_reason ||= example.failure&.message
+      @failure_reason ||= strip_invalid_utf8_chars(example.failure&.message)&.split("\n")&.first
```

Comment:
This is a bit of a nitpick so feel free to ignore, but there's a lot of lonely operators going on in this line, I dunno if its worth refactoring tho 🤷‍♀️